marissalmoser/Placebo-Entertainment
# Win lose screens

To test functionality of the win screen:
- Ensure there's a TabbedMenu prefab in your scene.
- In play mode, right click on the TabbedMenu component, and choose Test Win Screen On/Off.
- Observe the win screen.

To test functionality of the lose screen:
- Ensure there are TabbedMenu, LoopController, TimeManager, SaveManager prefabs in your scene.
- On the LoopController, set a loop time and delay time, such as 45 & 5. Weird things happen if this is not set.
- Go into play mode. Hit tab to view the schedule and look at the map to see the timer update. 
- Wait until the 30 second mark, the timer will appear on screen and the end sequence will play out.

Docs have been updated as well.
https://docs.google.com/document/d/1ydBjNoP09OcftJOvyShO_Gb-3rIgRBvGuSVhkGAXwo8/edit
